src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/draftorders/judgeapproval/JudgeApproval.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.draftorders.judgeapproval;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CaseDocument;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.YesOrNo;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.draftorders.CaseDocumentCollection;
+
+import java.util.List;
+
+import static java.util.Optional.ofNullable;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@Builder(toBuilder = true)
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class JudgeApproval {
+
+    private String title;
+
+    private String hearingInfo;
+
+    private YesOrNo hasAttachment;
+
+    @JsonProperty("document")
+    private CaseDocument document;
+
+    @JsonProperty("amendedDocument")
+    private CaseDocument amendedDocument;
+
+    @JsonProperty("judgeDecision")
+    private JudgeDecision judgeDecision;
+
+    @JsonProperty("attachments")
+    private List<CaseDocumentCollection> attachments;
+
+    @JsonProperty("isFinalOrder")
+    private YesOrNo isFinalOrder;
+
+    @JsonIgnore
+    private SortKey sortKey;
+
+    public YesOrNo getHasAttachment() {
```

Comment:
Should this have @JsonIgnore?

Reply:
From taking a look, yes we do need it. We only use this on the backend as an internal field for sorting JudgeApproval objects.

Reply:
From taking a look yes we do need it. We use this only for internal purposes.[c76b9fc](https://github.com/hmcts/finrem-case-orchestration-service/pull/1956/commits/c76b9fc0a7079c1a23d3af2f46b34a16a3ceeca9)